frontend/public/plugins.ts

```diff
@@ -30,13 +26,34 @@ export const initConsolePlugins = _.once((reduxStore: Store<RootState>) => {
   registerPluginEntryCallback(pluginStore);
 });
 
+const loadPluginFromURL = async (baseURL: string) => {
+  const manifest = await fetchPluginManifest(baseURL);
+  loadDynamicPlugin(baseURL, manifest);
+  return getPluginID(manifest);
+};
+
+// Load all dynamic plugins which are currently enabled on the cluster
+Promise.all(
```

Comment:
Do we need to call `Promise.all` if we aren't using the return value?

Reply:
We don't :smiley: loading a dynamic plugin is an async operation, but such operations are not dependent on one another so we can have a simple `window.SERVER_FLAGS.consolePlugins.forEach()` call instead. I'll make the change.

Reply:
Technically we could use `Promise.allSettled` which resolves after all plugins are either loaded or failed to load, but what we really want is to enable given plugin once it's loaded so a simple `then()` attached to each plugin's `Promise` should be sufficient.